admin-api-server/src/utils.ts

```diff
@@ -74,3 +75,25 @@ export const prepareFilterClause = (
   }
   return { query, params: Object.values(filter) };
 };
+
+
+export const prepareNftFilterClause = (
```

Comment:
This is an exact copy of `prepareFilterClause` right? And it's not applied. Think this function should be deleted.